src/js/components/AppPageComponent.jsx

```diff
@@ -182,6 +182,26 @@ var AppPageComponent = React.createClass({
     }
   },
 
+  getUnhealthyTaskMessage: function (healthCheckResults = []) {
+    return healthCheckResults.map((hc, index) => {
+      if (hc && !hc.alive) {
+        var failedCheck = this.state.app.healthChecks[index];
+
+        var protocol = failedCheck.protocol
+          ? `${failedCheck.protocol} `
+          : "";
```

Comment:
I agree with Pierlo to have a discussion again and creating well defined coding guidelines.
For ternaries I can imagine a way in the middle. I dislike them in markup, but in this special case here I find them pretty useful and don't see any disadvantage. But that are only my 2 Eurocents for now.
Let's go like it is in this PR and maybe have a bigger refactor task in the future at all based on our discussion results.